packages/components-react/paragraph-react/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@nl-design-system-candidate/paragraph-react",
+  "version": "1.0.0",
+  "description": "Displays a paragraph of text",
+  "license": "EUPL-1.2",
+  "private": true,
+  "type": "module",
+  "main": "./dist/paragraph.js",
+  "exports": {
+    ".": {
+      "types": "./dist/paragraph.d.ts",
+      "import": "./dist/paragraph.js"
+    }
+  },
+  "repository": {
+    "type": "git+ssh",
+    "url": "git@github.com:nl-design-system/candidate.git",
+    "directory": "packages/components-react/paragraph-react"
+  },
+  "scripts": {
+    "build": "tsc --project tsconfig.build.json",
+    "clean": "rimraf dist/ *.tsbuildinfo",
+    "test": "jest",
+    "typecheck": "tsc"
+  },
+  "devDependencies": {
+    "@types/react": "18.3.3",
+    "react": "18.3.1"
+  },
+  "peerDependencies": {
+    "react": "18.3.1"
```

Comment:
de react peer dependency hoeft niet een fixed version te zijn, ik denk dat het onhandig is voor mensen in de praktijk

Reply:
Ja goeie, die stond nog op het lijstje.

Reply:
Vanaf React 19 wordt `forwardRef` overbodig en kun je rechtstreeks de `ref` prop gebruiken maar hij wordt pas in een latere versie deprecated en verwijderd. Dus wel zullen wel `^18 || ^19` kunnen gaan gebruiken voor componenten.